src/cli/cms/editor/handlebars/printInput.js

```diff
@@ -336,10 +336,10 @@ export function printInput (params, root) {
   }
 
   if(params.value === null && params.default != null) params.value = params.default
-  if(params.forcedvalue != null) params.value = params.forcedvalue
+  if(params.forcedvalue != null && (params.value == null || params.value === '')) params.value = params.forcedvalue
```

Comment:
If you put an attribute "value" in your abe tag, this value will be ALWAYS used in the editor (hence in the json). 
If you put an attribute "default" in your abe tag, this value will be used in the editor ONLY the first time, during the creation of the document.
So This line shouldn't change.